src/main/kotlin/de/cyface/collector/handler/PreRequestHandler.kt

```diff
@@ -195,6 +205,7 @@ class PreRequestHandler(
      * @throws SkipUpload If the server is not interested in the data.
      * @throws Unparsable E.g. if there is a syntax error in the body.
      */
+    @SuppressWarnings("CyclomaticComplexMethod")
```

Comment:
Do not ignore this. It tells you, that the method is way too complex. It should be separated into methods who do exactly one thing.

Reply:
siehe FIXME Notizen in meinem lokalen Code